src/app/tool-editor/tool-editor.component.ts

```diff
@@ -119,6 +127,11 @@ import LoadOptions = jsyaml.LoadOptions;
                                (reset)="resetJob()"
                                [inputs]="toolModel.inputs"></ct-job-editor>
 
+                <ct-app-info *ngIf="viewMode === viewModes.Info"
+                              class="gui-editor-component p-2"
```

Comment:
Workflow info doesn't have this padding, it looks strange to have it on one and not the other. Add `p-1` to both.